### PR TITLE
Add ability to specify the min and max dates to the Calendar of Events

### PR DESCRIPTION
**Pull request by thomasgohard**
## What does this pull request (PR) do?
This PR adds the ability to specify the min and max dates to the Calendar of events to override the default behaviour of setting the date range based on the source list of events. This functionality is needed to support dynamically adding source events outside the range of existing events.

**General checklist / Liste de contrôle générale**

- [x] Create/update documentation /// Créer/mettre à jour la documentation
- [x] Build and test PR's code /// Rouler le script de compilation et tester le code de la PR
- [x] Validate changes against WCAG for accessibility /// Valider les changements avec WCAG pour l'accessibilité
- [x] Ensure documentation is bilingual /// S'assurer que la documentation soit bilingue

### CODE REVIEW

**Comment by duboisp** [comment]
Pre-appoved upon review, testing and once the build script is fixed.

**Review by duboisp — CHANGES_REQUESTED** [review]
Review and tested. It do work as expected and the additional feature is demoed. 

This change request is approved once the trailing spaces is removed as suggested.

File: src/plugins/cal-events/cal-events-fr.hbs [hunk 1]
```diff
@@ -486,6 +486,125 @@
...
 	&lt;/section&gt;
+&lt;/div&gt;</code></pre>
+			</details>
+		</section>
+	</section>
+
+	<section>
+		<h3>Plage de dates du calendrier</h3>
+		<p>Par défaut, la plage de dates du calendrier est fixée selon la plage de dates des événements sources. Ceci peut-être outrepassé en configurant les dates minimum et/ou maximum à l'aide des attributs <code>data-calevt-min-date</code> et <code>data-calevt-max-date</code>, qui prennent chacun une <a href="https://tc39.es/ecma262/multipage/numbers-and-dates.html#sec-date-time-string-format">chaîne de charactères de date et de temps en format AAAA-MM-JJ (lien disponible uniquement en anglais)</a>.</p>
+
```
[the remark this answers is not in the document]
Comment (thomasgohard) [reply src/plugins/cal-events/cal-events-fr.hbs]: Done.

**Review by duboisp — APPROVED** [review]

**Comment by duboisp** [comment]
We will merge it at our next technical review meeting, Jan 8

**Pull request merged**